project-base/storefront/components/Blocks/UserNavigation/UserNavigation.tsx

```diff
@@ -12,7 +13,7 @@ import { useUserMenuItems } from 'utils/user/useUserMenuItems';
 
 const getMenuItemTwClass = (menuItemLink?: string, asPath?: string) =>
     twJoin(
-        'border-b border-borderAccent last:border-none no-underline px-2 py-3 flex justify-between gap-4',
+        'border-b border-borderAccent last:border-none no-underline px-4 py-3 flex justify-between gap-4 text-sm',
```

Comment:
I understand that without a proper design, navigation aesthetics can feel subjective. However, I’d aim to keep it as close as possible to the current popup menu layout. It's entirely up to you whether to redesign or keep it as it is. Thanks anyway :)
![image](https://github.com/user-attachments/assets/f059dacd-3e2b-4c0f-8703-4d9fa08e0223)

Reply:
That was my first thought, but after i used it, it was too big so I ended up leaving it like this and I'd rather wait for a proper design